client/landing/jetpack-cloud/components/backup-delta/index.jsx

```diff
@@ -65,54 +70,179 @@ class BackupDelta extends Component {
 		);
 	}
 
+	renderMetaDiff() {
+		const { metaDiff, translate } = this.props;
+		const metas = [];
+
+		if ( metaDiff.plugins > 0 ) {
+			metas.push(
+				translate( '+%(numPlugins)d Plugin', '+%(numPlugins)d Plugins', {
+					count: metaDiff.plugins,
+					args: {
+						numPlugins: metaDiff.plugins,
+					},
+				} )
+			);
+		} else if ( metaDiff.plugins < 0 ) {
+			metas.push(
+				translate( '%(numPlugins)d Plugin', '%(numPlugins)d Plugins', {
+					count: metaDiff.plugins,
+					args: {
+						numPlugins: metaDiff.plugins,
+					},
+				} )
+			);
+		}
+
+		if ( metaDiff.themes > 0 ) {
+			metas.push(
+				translate( '+%(numThemes)d Theme', '+%(numThemes)d Themes', {
+					count: metaDiff.themes,
+					args: {
+						numThemes: metaDiff.themes,
+					},
+				} )
+			);
+		} else if ( metaDiff.themes < 0 ) {
+			metas.push(
+				translate( '%(numThemes)d Theme', '%(numThemes)d Themes', {
+					count: metaDiff.themes,
+					args: {
+						numThemes: metaDiff.themes,
+					},
+				} )
+			);
+		}
+
+		if ( metaDiff.uploads > 0 ) {
+			metas.push(
+				translate( '+%(numUploads)d Upload', '+%(numUploads)d Uploads', {
+					count: metaDiff.uploads,
+					args: {
+						numUploads: metaDiff.uploads,
+					},
+				} )
+			);
+		} else if ( metaDiff.uploads < 0 ) {
+			metas.push(
+				translate( '%(numUploads)d Upload', '%(numUploads)d Uploads', {
+					count: metaDiff.uploads,
+					args: {
+						numUploads: metaDiff.uploads,
+					},
+				} )
+			);
+		}
+
+		if ( metaDiff.posts > 0 ) {
+			metas.push(
+				translate( '+%(numPosts)d Post', '+%(numPosts)d Posts', {
+					count: metaDiff.posts,
+					args: {
+						numPosts: metaDiff.posts,
+					},
+				} )
+			);
+		} else if ( metaDiff.posts < 0 ) {
+			metas.push(
+				translate( '%(numPosts)d Post', '%(numPosts)d Posts', {
+					count: metaDiff.posts,
+					args: {
+						numPosts: metaDiff.posts,
+					},
+				} )
+			);
+		}
+
+		return <div className="backup-delta__metas">{ metas.join( ', ' ) }</div>;
+	}
+
 	renderDaily() {
-		const { backupAttempts, deltas, translate } = this.props;
+		const { backupAttempts, deltas, metaDiff, translate } = this.props;
 		const mainBackup = backupAttempts.complete && backupAttempts.complete[ 0 ];
-		const meta = mainBackup && mainBackup.activityDescription[ 2 ].children[ 0 ];
 
-		const media = deltas.mediaCreated.map( item => (
-			<div key={ item.activityId }>
+		const mediaCreated = deltas.mediaCreated.map( item => (
+			<div key={ item.activityId } className="backup-delta__media-image">
 				<img alt="" src={ item.activityMedia.thumbnail_url } />
-				<div>{ item.activityMedia.name }</div>
+				<div className="backup-delta__media-title">
+					<Gridicon icon="add" />
+					{ translate( 'Added' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **80** times:
`translate( 'Added:' )` *ES Score: 7*